hack/build-libgit2.sh

```
@@ -0,0 +1,15 @@
+
+sudo apt-get update
+sudo apt-get install -y pkg-config
+sudo apt update && \
+sudo DEBIAN_FRONTEND=noninteractive apt install -y build-essential cmake pkg-config libssl-dev libssh2-1-dev zlib1g-dev libhttp-parser-dev python3 wget tar git && \
+wget https://github.com/libgit2/libgit2/archive/refs/tags/v1.5.1.tar.gz -O libgit2-v1.5.1.tar.gz && \
+tar -xzf libgit2-v1.5.1.tar.gz && \
+cd libgit2-1.5.1 && \
+mkdir build && \
+cd build && \
+cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF && \
+make -j$(nproc) && \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)